examples/secret_channel.py

```diff
@@ -0,0 +1,106 @@
+# import discord and any other libraries
+import discord
+from discord.ext import commands
+from discord.utils import get
+
+# bot description
+description = "A simple example showcasing how to make text and voice channels (as well as private ones)."
+
+# create a bot instance with prefix "?" and description
+bot = commands.Bot(command_prefix="?", description=description)
+
+@bot.event
+async def on_ready():
+    """Prints information to the console once the bot is ready."""
+
+    print('Logged in as')
+    print(bot.user.name)
+    print(bot.user.id)
+    print('Bot Online!')
+
+def can_create():
```

Comment:
This entire check already comes built-in, it's `commands.has_permissions(manage_channels=True)`